kyototycoon/tests/test_kyototycoon.py

```diff
@@ -3,43 +3,50 @@
 # Licensed under a 3-clause BSD style license (see LICENSE)
 from copy import deepcopy
 
-from six import itervalues
+import pytest
 
 from datadog_checks.kyototycoon import KyotoTycoonCheck
 
 from .common import DEFAULT_INSTANCE, TAGS
 
+GAUGES = list(KyotoTycoonCheck.GAUGES.values())
+DB_GAUGES = list(KyotoTycoonCheck.DB_GAUGES.values())
+TOTALS = list(KyotoTycoonCheck.TOTALS.values())
+RATES = list(KyotoTycoonCheck.RATES.values())
+
+# all the RATE type metrics
+ALL_RATES = TOTALS + RATES
+
 
 def test_check(aggregator, dd_environment):
-    """
-    Testing Kyototycoon check.
-    """
     kt = KyotoTycoonCheck('kyototycoon', {}, {})
-
-    # run the check
     kt.check(deepcopy(DEFAULT_INSTANCE))
+    kt.check(deepcopy(DEFAULT_INSTANCE))
+
+    _assert_check(aggregator, 2)
+
+
+@pytest.mark.e2e
+def test_e2e(dd_agent_check):
+    aggregator = dd_agent_check(DEFAULT_INSTANCE, rate=True)
 
-    GAUGES = list(itervalues(KyotoTycoonCheck.GAUGES))
-    DB_GAUGES = list(itervalues(KyotoTycoonCheck.DB_GAUGES))
-    TOTALS = list(itervalues(KyotoTycoonCheck.TOTALS))
-    RATES = list(itervalues(KyotoTycoonCheck.RATES))
+    _assert_check(aggregator, 1)
 
-    # all the RATE type metrics
-    ALL_RATES = TOTALS + RATES
 
+def _assert_check(aggregator, rate_metric_count):
```

Comment:
```suggestion
def _assert_check(aggregator, rate_metric_count=1):
```

I would suggest having a default value of one, removing the argument in `test_e2e` and naming it in `test_check`
This makes it easier to understand than `_assert_check(aggregator, 1)`

Reply:
Sure, fixed :)